php/Terminus/User.php

```diff
@@ -73,12 +75,21 @@ public function sites($organization=null) {
     return $response['data'];
   }
 
-  public function getId() {
-    return $this->id;
+  public function instruments() {
+    $instruments       = new Instruments(array('user' => $this));
```

Comment:
> Why have a useless collection?

My perspective is that one invariant of a `User` instance is that they have a collection of `Instruments`. I don't care about the internal state of `Instruments` (whether it has been fetched or not), but I expect that it be there throughout the entire lifecycle of the instance. This is essentially the [NullObject Pattern](https://en.wikipedia.org/wiki/Null_Object_pattern) but we don't need to create a different kind of object, we can just attach the empty collection.